ast.go

```diff
@@ -246,47 +230,65 @@ func (node *Subquery) ContainsSubquery() bool {
 	return true
 }
 
-// ParenTableExpr represents a parenthesized list of TableExpr.
+// ParenTableExpr represents a parenthesized TableExpr.
 type ParenTableExpr struct {
-	TableExprList TableExprList
+	TableExpr TableExpr
 }
 
 // String returns the string representation of the node.
 func (node *ParenTableExpr) String() string {
-	return fmt.Sprintf("(%v)", node.TableExprList.String())
+	return fmt.Sprintf("(%v)", node.TableExpr.String())
+}
+
+// JoinOperator represents a join operator.
```

Comment:
Before the `JOIN` operator was represented as a simple `string`. Now it has its own type. [See](https://github.com/tablelandnetwork/go-sqlparser/pull/33/files#diff-0945956cd950d047e7fa866dcf34fee7d29cbce277d4e5e5cd621f2213f3be6cR265).